Prefetching-Library/android_prefetching_lib/src/main/java/nl/vu/cs/s2group/nappa/prefetch/PrefetchingStrategy.java

```diff
@@ -15,78 +16,13 @@
  * library API via:
  * <br/><br/>
  *
- * <code>PrefetchingLib.init(STRATEGY_ID)</code>
+ * <code>PrefetchingLib.init(PrefetchingStrategyType.STRATEGY_ID)</code>
  * <br/><br/>
  *
- * <p>The available values of the {@code STRATEGY_ID} parameter are in the following list:
- *
- * <ul>
- *     <li> {@link #STRATEGY_MOST_VISITED_SUCCESSOR}</li>
- *     <li> {@link #STRATEGY_2}</li>
- *     <li> {@link #STRATEGY_GREEDY}</li>
- *     <li> {@link #STRATEGY_PPM} (Prediction by Partial Match)</li>
- *     <li> {@link #STRATEGY_PAGERANK}</li>
- *     <li> {@link #STRATEGY_HITS} (Hyperlink-Induced Topic Search)</li>
- *     <li> {@link #STRATEGY_SALSA} (Stochastic Approach for Link-Structure Analysis)</li>
- *     <li> {@link #STRATEGY_GREEDY_WITH_PAGERANK_SCORES}</li>
- *     <li> {@link #STRATEGY_PPM_WITH_HITS_SCORES}</li>
- * </ul>
- *
```

Comment:
Moved to an enumerated type.
Caught this via Sonar Cloud and indeed, using an enum for this is better.